packages/suite-desktop-core/e2e/support/fixtures.ts

```diff
@@ -37,24 +38,42 @@ const test = base.extend<Fixtures>({
     trezorUserEnvLink: async ({}, use) => {
         await use(TrezorUserEnvLink);
     },
-    electronApp: async ({ trezorUserEnvLink, startEmulator, emulatorConf }, use) => {
+    appContext: async (
+        { trezorUserEnvLink, startEmulator, emulatorConf, locale, colorScheme },
+        use,
+        testInfo,
+    ) => {
         // We need to ensure emulator is running before launching the suite
         if (startEmulator) {
             await trezorUserEnvLink.stopBridge();
+            await trezorUserEnvLink.stopEmu();
             await trezorUserEnvLink.connect();
             await trezorUserEnvLink.startEmu({ wipe: true });
             await trezorUserEnvLink.setupEmu(emulatorConf);
         }
-        const suite = await launchSuite();
-        await use(suite.electronApp);
```

Comment:
what is this `use` thingy? I don't even remember this code

Reply:
That is playwright function that allow you to provide fixtures to tests or other fixtures. The fixture can be any asset that you prepare, either page object model class or extension of default page object with your own custom methods

So in test for example. you can import two already initilized fixtures `onboardingPage`, `settingsPage` and do whatever you need. Keeping the test clearer and more easy to read. 
```
    test.beforeEach(async ({ onboardingPage, settingsPage }) => {
        await onboardingPage.disableFirmwareHashCheck();
        await onboardingPage.completeOnboarding();
        await settingsPage.navigateTo();
        await settingsPage.deviceTabButton.click();
    });
```